### PR TITLE
HOTFIX: fix defusedxml installation.

### PR DESCRIPTION
**Pull request by ernstleierzopf**
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [ ] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [ ] Issues exist for this PR
- [ ] I added related issues using the "Fixes #<issue-id>"-notations
- [ ] This Pull-Requests merges into the "development"-branch

Fixes #

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

# Describe changes:
-

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -45,6 +45,9 @@ RUN apt-get update && apt-get install -y \
     python3-urllib3 \
     python3-statsmodels \
     python3-pandas \
+    python3-numpy \
+    python3-patsy \
```
Comment (whotwagner) [inline Dockerfile:49]: Do we need patsy?

File: aecid-testsuite/Dockerfile [hunk 2]
```diff
@@ -47,7 +47,7 @@ RUN apt-get update && apt-get install -y \
     python3-pylibacl \
     python3-urllib3 \
     python3-statsmodels \
-    python3-pandas \
```
Comment (whotwagner) [inline aecid-testsuite/Dockerfile:50]: Don't we need pandas anymore?

File: Dockerfile (hunk 1 above)
Comment (ernstleierzopf) [reply Dockerfile:48]: yes we do, my mistake.

Comment (ernstleierzopf) [reply Dockerfile:48]: yes we do.
from statsmodels.tsa.arima.model import ARIMA
  File "/etc/aminer/conf-enabled/statsmodels/tsa/arima/model.py", line 14, in <module>
    from statsmodels.tsa.statespace import sarimax
  File "/etc/aminer/conf-enabled/statsmodels/tsa/statespace/sarimax.py", line 19, in <module>
    from statsmodels.tsa.arima.specification import SARIMAXSpecification
  File "/etc/aminer/conf-enabled/statsmodels/tsa/arima/specification.py", line 11, in <module>
    from statsmodels.tsa.base.tsa_model import TimeSeriesModel
  File "/etc/aminer/conf-enabled/statsmodels/tsa/base/tsa_model.py", line 26, in <module>
    import statsmodels.base.model as base
  File "/etc/aminer/conf-enabled/statsmodels/base/model.py", line 12, in <module>
    from statsmodels.formula import handle_formula_data
  File "/etc/aminer/conf-enabled/statsmodels/formula/__init__.py", line 2, in <module>
    from .formulatools import handle_formula_data
  File "/etc/aminer/conf-enabled/statsmodels/formula/formulatools.py", line 2, in <module>
    from patsy import dmatrices, NAAction
ModuleNotFoundError: No module named 'patsy'

File: aecid-testsuite/Dockerfile (hunk 2 above)
Comment (ernstleierzopf) [reply aecid-testsuite/Dockerfile:50]: we need pandas, my mistake.

**Comment by ernstleierzopf** [comment]
fixed everything now; PR can be accepted.

**Review by landauermax — APPROVED** [review]
Looks good to me

**Pull request merged**